docs/stake-transition.md

```diff
@@ -0,0 +1,147 @@
+# Transition of Existing Stakes to the Babylon Chain
+
+This document is intended for Bitcoin stakers who have
+submitted confirmed Bitcoin staking transactions
+on the Bitcoin blockchain before the launch of
+the phase-2 Babylon blockchain (e.g., via participation
+in one of the phase-1 networks).
+It outlines the eligibility criteria for the registration of
+the stakes, as well as the required data to submit to the
+Babylon blockchain to gain voting power and start earning
+rewards.
+
+> **⚡ Note:**: Each Babylon blockchain network is associated
+> with a Bitcoin network (e.g.,
+> test environments connect to the BTC signet,
+> while production connects to the BTC mainnet).
+> For the remainder of the document, we will
+> assume that you are registering a stake that
+> has been created on the corresponding Bitcoin network
+> with the expected parameters (e.g.,
+> for phase-2 testnet, registering stakes from phase-1 testnet).
+
+## Registration Eligibility
+
+Bitcoin stakers can hold multiple stakes. The following
+flowchart outlines the criteria for each stake's transition.
+Note that eligibility is assessed on a
+*per-stake* basis, meaning that some stakes
+may be eligible for registration while others are not.
+
+![Stake Registration Eligibility](./static/stake-register-eligibility.png)
+
+The following subsections explain the different
+conditions presented in the diagram.
+
+#### Stake Validity
+
+A stake is valid and eligible for registration on the Babylon blockchain if
+it meets the following criteria:
+1. **Confirmed on the Bitcoin ledger**:
+   The staking transaction must have received sufficient
+   Bitcoin confirmations to be considered final.
+   The Babylon blockchain defines this in its
+   [module parameters](../proto/babylon/btccheckpoint/v1/params.proto)
+   (expected to be set to 10 confirmations for testnets and mainnet).
+2. **Compliance with Staking Parameters**:
+   Staking transactions must adhere to the Bitcoin block-range parameters
+   defined by the Babylon blockchain, which are consistent with those of
+   the Phase-1 system (e.g., covenant keys, and minimum and maximum stake requirements).
```

Comment:
```suggestion
   the Phase-1 system (e.g., covenant keys, and minimum and maximum stake requirements).
```
We are using both `Phase-1` and `phase-1`. Should we stick with `Phase-1` and `Phase-2`?